simulator/src/main/java/com/hazelcast/simulator/coordinator/CoordinatorCli.java

```diff
@@ -122,8 +122,9 @@
                     + " Use ';' as separator for multiple entries. The wildcard '*' can also be used.")
             .withRequiredArg().ofType(String.class);
 
-    private final OptionSpec<String> testSuiteIdSpec = parser.accepts("testSuiteId",
-            "Defines the ID of the TestSuite. If not set the actual date will be used.")
+    private final OptionSpec<String> sessionIdSpec = parser.accepts("sessionIdSpec",
```

Comment:
I think the option itself should be "sessionId" instead of "sessionIdSpec". Also there is a missing space in the description. Fixed both.